SQL/paradise_schema.sql

```diff
@@ -446,10 +446,13 @@ DROP TABLE IF EXISTS `erro_watch`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `erro_watch` (
-  `id` int(11) NOT NULL AUTO_INCREMENT,
   `ckey` varchar(32) NOT NULL,
   `reason` text NOT NULL,
-  PRIMARY KEY (`id`)
+  `timestamp` datetime NOT NULL,
+  `adminckey` varchar(32) NOT NULL,
```

Comment:
These table changes don't really make much sense, and make it less consistant. Instead of using the auto incrementing id as every other table does for a primary key, it uses the ckey, which doesn't make much sense at all.